src/tiler/server.js

```diff
@@ -103,9 +103,11 @@ var config = {
             }
 
             if (body) {
-                s3Obj
-                    .upload({Body: body})
-                    .send(function(err, data) { throw (err); });
```

Comment:
Multiple tile requests at a time was causing this to fail with a `Cannot read property 'message' of null` deep in the AWS SDK. I messed around with updating the SDK in case it was a bug, but that didn't resolve it. After tweaking the way the function was written the errors went away. Was unsure about what `send()` was doing here.